datafusion/physical-expr-common/Cargo.toml

```diff
@@ -39,3 +39,4 @@ path = "src/lib.rs"
 arrow = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+sqlparser = { workspace = true }
```

Comment:
I think ideally the physical expr shouldn't be depending on sql parser (for people who are not using SQL)

Though now I see that perhaps it is because AggregateFunction has a null treatment flag on it 🤔  That might be a nice dependency to avoid

Reply:
I think it is easy to avoid this dependency, we can check this token in parser, then pass boolean all the way down.

Reply:
let me file an PR for this